FogCreek/Glitch-Community
# Coordinated mascara: componentizing note

## Links
* https://coordinated-mascara.glitch.me/@sarahzinger/another-collection
* https://glitch.manuscript.com/f/cases/3328271/
* related bug fix: https://glitch.manuscript.com/f/cases/3328463/Add-style-for-MarkdownInput-to-render-with-editable-font-when-the-user-is-authorised

## GIF/Screenshots:
<img width="1096" alt="Screen Shot 2019-04-25 at 6 01 03 PM" src="https://user-images.githubusercontent.com/6620164/56771244-22480500-6784-11e9-8a10-18536e34cf07.png">

## How To Test:
* check out storybook and look for the note section
* go to a collection and create a note
* go to a collection, feature a project, add a note to that featured project
* go to a collection, hide a note

## Feedback I'm looking for:
* double check there aren't any regressions here I'm missing
* I left a lot of the direct tag styles (overwriting textarea and stuff) as that's how we had it and it all just seemed to work? But is this desirable? Is there another way to do this? Should I go into these markdown inputs and give them a type of note and pass that to them maybe?
* there are probably some general style bugs with this component as well, I may or may not get to them in this pr but if you see em let me know and I'll make sure to fix them here or make a case for it. For example I know [placeholder text and underline colors are still not always enough contrast depending on the background color](https://glitch.manuscript.com/f/cases/3328432/). 

## Things left to do before deploying:
- [ ] I'd like to give these styles another double check, to make sure they are all necessary and we haven't missed anything. 
- [ ] crossbrowser test
